device.go

```diff
@@ -253,7 +258,12 @@ func GetHostPath(devInfo DeviceInfo) (string, error) {
 
 	content, err := ini.Load(sysDevPath)
 	if err != nil {
-		return "", err
+		// Some devices(eg. /dev/fuse, /dev/cuse) do not always implement sysfs interface under /sys/dev
+		// These devices are passed by default by docker.
+		//
```

Comment:
Not erroring out on `ini.Load()` failure could hide some other issues. Can we simply return `devInfo.ContainerPath, nil` when `sysDevPath` does not exist?

Reply:
+1